editor/static/css/editor.css

```
@@ -125,61 +131,640 @@
+    padding-bottom: 2vh !important;
+    padding-left: 2vw !important;
+    padding-right: 2vw !important;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    background: rgba(255, 255, 255, 0.95);
+    backdrop-filter: blur(10px);
+    border: 1px solid var(--secondary-color);
+    border-radius: var(--sb-border-radius-md);
+    padding: 6px 10px;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)